native/explorer/src/expressions.rs

```diff
@@ -57,6 +57,15 @@ pub fn expr_date(date: ExDate) -> ExExpr {
     ExExpr::new(expr)
 }
 
+#[rustler::nif]
+pub fn expr_time(_time: ExTime) -> ExExpr {
+    // FIXME: lit() is not available on NaiveTime
```

Comment:
I'm not sure if this is possible. We may need to ask Polars team to add (or we open a PR there) this.
See: https://pola-rs.github.io/polars/polars/prelude/trait.Literal.html